src/hotspot/share/oops/klassVtable.cpp

```diff
@@ -1094,8 +1094,8 @@ void itableMethodEntry::initialize(InstanceKlass* klass, Method* m) {
 #ifdef ASSERT
   if (MetaspaceShared::is_in_shared_metaspace((void*)&_method) &&
      !MetaspaceShared::remapped_readwrite() &&
-     !m->method_holder()->can_be_verified_at_dumptime() &&
-     !klass->can_be_verified_at_dumptime()) {
+     m->method_holder()->verified_at_dump_time() &&
+     klass->verified_at_dump_time()) {
```

Comment:
This change seems to go beyond the simple logic negation. Why the change here?

Reply:
Hi David,

Thanks for your review!

The block of code you mentioned is being called during runtime. So it is more consistent to call `verified_at_dump_time()`.

Thanks,
Calvin